.github/workflows/gerrit-ci-management-verify.yaml

```diff
@@ -134,6 +134,10 @@ jobs:
         # yamllint disable rule:line-length
         run: |
           jjb_version="${JJB_VERSION:-5.1.0}"
+          version_line=$(grep '^jenkins-job-builder==' "./requirements.txt")
+          version_substring=$(echo "${version_line}" | cut -d '=' -f 3)
+          version_value=$(echo "${version_substring}" | cut -d ' ' -f 1 | cut -d '#' -f 1)
+          jjb_version="${version_value:-$jjb_version}"
```

Comment:
All this might be a bit of a overkill, since its easier to set the JJB version env var in the downstream workflow file.
in FDio repo (ci-man workflow file).

https://github.com/lfit/releng-reusable-workflows/blob/main/.github/workflows/gerrit-ci-management-verify.yaml#L131-L136

Reply:
The idea was that there shall be only one source of truth (for JJB version), in my case that would be ci-management repo. That way, single Gerrit patch to ci-management will cause all verify jobs to apply the new JJB version when verifying it.
Having one change for ci-management plus one PR for github.com/FDio/.github is harder to verify together.

That is why github.com/FDio/.github should be very minimalistic so it is updated only rarely, with all possibly movable logic living in ci-management.